test/tests/props.js

```diff
@@ -82,7 +82,56 @@ describe('zoid props cases', () => {
                     }
                 })
             });
-            
+
+            return instance.render(getBody());
+        });
+    });
+
+    it('should merge props definition with input props before passing to decorator', () => {
+        return wrapPromise(({ expect }) => {
+            window.__component__ = () => {
+                return zoid.create({
+                    tag:    'test-prop-value-container-url-param-bug',
+                    url:    'mock://www.child.com/base/test/windows/child/index.htm',
+                    domain: 'mock://www.child.com',
+                    props:  {
+                        account: {
+                            type:       'string',
+                            queryParam: false,
+                            required:   true,
+                            value:      ({ props }) => {
+                                return props.account;
+                            }
+                        },
+                        testAccountParam: {
```

Comment:
I like how messaging-components describes this pattern as computed props ([link](https://github.com/paypal/paypal-messaging-components/blob/f0cd8d1e9b6dce5ad4602ad559e3f4f4824f10d5/src/zoid/message/component.js#L313)). Can we add a comment here that describes this behavior? Ex:

```js
    // computed property that depends on the `account` prop
   testAccountParam: {
```

Reply:
Yeah for sure. Just to be clear are you thinking of adding information about that pattern to the docs or do you mean just some comments in the test to describe it? I'm happy to both if we think thats the right idea